store/kv/measure.go

```diff
@@ -58,14 +110,27 @@ func (m *TxStoreWithMetrics) CreateTable(ctx context.Context, name []byte) (err
 		err = m.kv.CreateTable(ctx, name)
 		return err
 	})
+	setSingleWriteUnit(ctx)
 	return
 }
 
 func (m *TxStoreWithMetrics) DropTable(ctx context.Context, name []byte) (err error) {
+	size, err := m.kv.TableSize(ctx, name)
```

Comment:
let's emit a different metric for drop and we don't need to calculate size.

Reply:
Should it be single write unit?

Reply:
I will change this to Drop Unit.